Se 13419 rtm infinite link loop hn (#216)
mulesoft/docs-runtime-manager: modules/ROOT/pages/servers-cert-renewal.adoc

menu= Renew Server Certificates (Hybrid)
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]

> Register a Mule server in Runtime Manager requires a valid certificate provisioned and signed by Runtime Manager. The certificate ensures secure communications between Runtime Manager and the Runtime Manager agent. Before a certificate expires, you can renew it directly from Runtime Manager. You cannot renew a certificate after it expires. You can update a certificate for a specific server or multiple servers within a cluster or group.

Reviewer: ```suggestion
Registering a Mule server in Runtime Manager requires a valid certificate provisioned and signed by Runtime Manager. The certificate ensures secure communications between Runtime Manager and the Runtime Manager agent. 

However, certificates expire, and you cannot renew them after they expire. Before a certificate expires, renew it directly from Runtime Manager. You can see the expiration date in the server dashboard.

A server must be running for you to update its certificate. You can update a certificate for a specific server or multiple servers within a cluster or group.
```
QUESTION about the final sentence: By "update," do we mean "renew"? Clarify the difference, if there is one.

> The expiration date of the certificate appears in the server dashboard. 

Reviewer: OMIT; added above.
```suggestion
```

To renew a certificate for a server:

. From Anypoint Platform, select *Runtime Manager*.
. Click the *Servers* tab.

> . Verify that the server status is *Running*.

Reviewer: ditto status format

+

> The server must be running to renew the certificate.

Reviewer: Do we need to say this, given the step?
Actually...I have suggested adding this to the intro text above.

. Click the server name.
. Click *Settings*.
. From the *Actions* menu, select *Renew Certificate*.

> . Click the checkbox to confirm, then click *Renew*.

Reviewer: ```suggestion
. Click the checkbox to confirm your choice, and then click *Renew*.
```
